app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -36,6 +36,8 @@ class Api::V2::RepositoriesController < Api::V2::ApiController
 
     api :GET, "/repositories", N_("List of enabled repositories")
     api :GET, "/content_views/:id/repositories", N_("List of repositories for a content view")
+    api :GET, "/products/:id/repositories", N_("List of repositories for a product")
+    api :GET, "/environments/:id/products/:id/repositories", N_("List of repositories belonging to a product in an environment")
```

Comment:
I think this might should be the parameter, in this case `:environment_id` and `product_id` ? In order for hammer and the docs to parse it correctly.